app/routes.py

```diff
@@ -55,12 +57,20 @@ async def random_sleep(response: Response):
     return {"path": "/random_sleep"}
 
 
-@router.get("/info_test")
-async def info_test(response: Response):
-    logger.info("got error!!!!")
+@router.get("/unhandled_exception")
+async def unhandled_exception(response: Response):
+    logger.error("got unhandled exception")
     raise ValueError("value error")
```

Comment:
FastAPI doesn't handle this. Instead what happens is it becomes an unhandled exception and in a server environment you end up with an HTTP 500, but in a test environment the exception is unhandled. I adjusted the text to make it clearer that's what we're going for in this endpoint.